CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   included in the Phusion Ruby base image for the DAP appliance
   [cyberark/conjur-base-image#10](https://github.com/cyberark/conjur-base-image/pull/10)
 
+- ARM64 image build is added
```

Comment:
1. this needs a rebase, because we've had a release since you last posted this
2. following the syntax of the other changelog entries, this should be more like:

> - Support for building ARM64 images to enable running the Conjur server on ARM devices like Raspberry Pi or newer Macbooks.
> [cyberark/conjur#1988](https://github.com/cyberark/conjur/issues/1988)